spk/sickchill/Makefile

```diff
@@ -1,20 +1,19 @@
 SPK_NAME = sickchill
-SPK_VERS = $(shell date +%Y%m%d)
+SPK_VERS = 20210419
 SPK_REV = 1
 SPK_ICON = src/sickchill.png
 
-DEPENDS = cross/$(SPK_NAME)
 SPK_DEPENDS = "python3>=3.6:git"
```

Comment:
@publicarray can we do `python3>=3.6||python>=3.6:git` here? Or is there some other reason Synology official python won't satisfy the dependency?

Reply:
No we can't. We must decide for one or the other (or create two different packages like sickchill/Sickchill (or sc-sickchill/synology-sickchill), but as git package is by synocommunity I would prefere to keep it like `"python3;git"`

Reply:
Yeah it wasnt the git package I was worried about, it is the confusion by some people who already have python 3 installed from syno official and then are asked to install python3 from sc when they install sickchill. One is named python, the other is named python3, even though both are python3.6+

Reply:
As a rule of thumb the (internal) name of packages from SynoCommunity are all lower case and the packages from synology start with uppercase letter (synocommunity: mono, synology: Mono)